src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -105,4 +105,103 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$attendee" missing
* :no_entry_sign: Doc comment for parameter "$ticket" missing
* :no_entry_sign: Doc comment for parameter "$attendee_data" missing

<sub>hash: ffe38dada3a8997e25b350b05dfba3cb</sub>